packages/d3fc-webgl/src/symbolMapper.js

```diff
@@ -0,0 +1,20 @@
+import d3Shape from 'd3-shape';
+import circlePointShader from './shaders/point/circle/baseShader';
+import squarePointShader from './shaders/point/square/shader';
+
+export default (symbol) => {
```

Comment:
This block will simplify down a fair bit if you just return the value in each case rather than `glSymbol = x; return;`